pkgs/development/python-modules/hvac/default.nix

```
@@ -9,14 +9,14 @@
 
 buildPythonPackage rec {
   pname = "hvac";
-  version = "1.2.1";
+  version = "2.0.0";
   format = "pyproject";
 
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/hvac/hvac/blob/91610440581bbb543f1eff02650f1a996f263e4c/pyproject.toml#L37